corpusreaders/src/main/java/edu/illinois/cs/cogcomp/nlp/corpusreaders/AbstractIncrementalCorpusReader.java

```diff
@@ -30,8 +31,12 @@
  * annotations as soon as last TextAnnotation from current file is returned.
  */
 public abstract class AbstractIncrementalCorpusReader extends TextAnnotationReader {
-    // list of all files that comprise corpus
-    private List<Path> fileList;
+    /**
+     * contains pointers to files comprising corpus.  Each entry may consist of a source document (first element
+     *    of each list) plus zero or more files with related annotations needed to instantiate the
+     *    corresponding TextAnnotation objects.
+     */
+    private List<List<Path>> fileList;
 
     // holds TextAnnotations extracted from most recently accessed file
```

Comment:
I notice the field above has a corrected JavaDoc comment included(thanks Mark), I suggest if we do it for one, we do it for all. In fact, we should force a JavaDoc style sheet. That's another conversation.